nova/virt/vmwareapi/vmops.py

```diff
@@ -1899,7 +1908,8 @@ def migrate_disk_and_power_off(self, context, instance, dest,
                                        total_steps=RESIZE_TOTAL_STEPS)
 
         # 3.Reconfigure the disk properties
-        self._resize_disk(instance, vm_ref, vmdk, flavor)
+        if (not cinder_boot):
```

Comment:
You only need `( )` in python's `if`, if you have multi-line conditions. Please don't use them otherwise.

Reply:
ok, bad habit from C++ and C#